doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_cf_4_7_0.adoc

```diff
@@ -26,7 +26,7 @@ CFME 5.10.4 does not support migration.
 endif::rhv[]
 ifdef::osp[]
 [[Cloudforms_for_osp]]
-. Install Red Hat CloudForms 4.7.3 (CFME 5.10.3). See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/[Installing Red Hat CloudForms on Red Hat OpenStack Platform].
+. Install Red Hat CloudForms 4.7.0 (with CFME 5.10.3 virtual appliance). See link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/[Installing Red Hat CloudForms on Red Hat OpenStack Platform].
```

Comment:
Same comment as above - 4.7.0/5.10.3?

Reply:
Probably less relevant for OSP. I'll ask.